app/facades/order/order_facade.rb

```diff
@@ -0,0 +1,31 @@
+class Order
+  class OrderFacade
+    def initialize; end
+
+    def orders
+      @orders ||= menu.orders
+    end
+
+    def meals(order)
+      order.meals.map { |m| "#{m.course}: #{m.name}" }
```

Comment:
Extract `"#{m.course}: #{m.name}"` to a separate method. Haven't you read Sandi Matz? 😛